docs/about/overview/encryption.mdx

```diff
@@ -19,6 +19,72 @@ All periodic broadcasts (position, telemetry, etc.) the device sends out itself
 
 The device will decrypt the payload before sending it to a client app via BLE, serial, Wi-Fi/Ethernet. For MQTT you can [specify](/docs/configuration/module/mqtt#encryption-enabled) whether you want to send an encrypted or unencrypted payload.
 
+## Is it as secure as Wi-Fi WPA3, HTTPS TLS1.3 or Signal ?
+
+**No**.
+
+[WPA3](https://en.wikipedia.org/wiki/WPA3), [TLS1.3](https://en.wikipedia.org/wiki/TLS1.3), [Signal](https://en.wikipedia.org/wiki/Signal_Protocol) and Meshtastic can all use AES256, yet AES is one of the many cogs required in a modern encryption system, see below the main features "missing" in Meshtastic.
+
+### Perfect-Forward-Secrecy
+
+[**P**erfect-**F**orward-**S**ecrecy also known as **PFS**](https://en.wikipedia.org/wiki/Forward_secrecy) means *previous* communications and more generally anything *passively captured* cannot be decrypted ***even when the key is compromised***.
+
+This means Meshtastic is vulnerable to [**« Harvest now, Decrypt later »**](https://en.wikipedia.org/wiki/Harvest_now,_decrypt_later) attacks, **this attack itself is not enough to decrypt messages** it relies on an other failure leaking the channel key such as:
+- Accidently sharing the channel key with the wrong person.
+- One of your node being stolen.
+- Any unknown bug revealing the channel key to an attacker.
+
+Other protocols such as *missconfigured TLS1.2* are far more impacted by Harvest now, Decrypt later attacks because their keys will be broken by near future Quantum-Computers yet they do not provide PFS.
+
+Meshtastic's encryption is not threaten by Quantum-Computers\* so realistic attack vectors are accidental miss-handling of channel keys and nodes being lost.
+
+\*on the Quantum-Resistance of AES256 see:
+- [this stackoverflow question, contain a bit of debates and two good answers](https://crypto.stackexchange.com/q/6712)
+- [NIST's Post-Quantum FAQ](https://csrc.nist.gov/projects/post-quantum-cryptography/faqs) section « To protect against the threat of quantum computers, should we double the key length for AES now? (added 11/18/18) »
+
+**Recomendations** for users using *private channels*:
+- **Do not configure private channels on unattended nodes**, nodes will relay meshtastic traffic even if they are not able to decrypt it.
+  Your unattended routers should not have the private PSK configured as it is easy gain physical access and extract the channel key.
+- Keep in mind, everything sent on a channel can be stored and decrypted later by anyone who gain access to the key even if you locally delete the messages.
+- Change your Channel Keys from time to time.
+
+### Integrity
+
+Integrity means ensuring messages cannot be modified without the key.
+
+Meshtastic does not check messages have not been tempered with, see [`#4030`](https://github.com/meshtastic/firmware/issues/4030) for details.
+
+**Recomendations**:
+- Keep in mind, anyone could send messages on a channel even with the channel key is private and secure.
```

Comment:
I'm not entirely following this. Yes, you can send a packet to a channel, but if you don't know the proper key to encrypt it, the node decrypting it would receive it as garbage.

Reply:
> I'm not entirely following this. Yes, you can send a packet to a channel, but if you don't know the proper key to encrypt it, the node decrypting it would receive it as garbage. It (highly likely) won't be a properly encoded protobuf message, so the node would discard it.

This is a known plaintext attack. if you can derive the cipher stream you can impersonate a node up to the length of the cipher stream you discovered. No need to have the proper key.

Reply:
Ah, this was referring to the attack. Currently the text reads as if anyone can do this by default. I would suggest to modify the text by adding "by applying a known plaintext attack...", or similar.